manifests/base/runner-manager/redis-config.env

```diff
@@ -0,0 +1 @@
+REDIS_OM_URL=redis://redis:6379
```

Comment:
Just realized something here, doesn't have to be now, but we'll have to figure out, eventually, how to transform the redis service name here, I believe that something like this should be set:

```yaml
# kustomization.yaml

replacements:
- source:
    kind: Service
    name: redis
  targets:
    - select:
        kind: ConfigMap
        name: redis-config
      fieldPaths:
        - data.REDIS_OM_URL
      options:
        delimiter: /
        index: 2 # might be 1 if the empty one is ignored
    
```

This is also may mean we might have to move this file into the redis folder and set those inside the `manifests/base/redis/kustomization.yaml` file